common/database/database_update_manifest.cpp

```diff
@@ -4745,6 +4745,24 @@ ADD COLUMN `id` int(11) NOT NULL AUTO_INCREMENT FIRST,
 ADD PRIMARY KEY (`id`);
 )",
 	},
+	ManifestEntry{
+		.version = 9229,
+		.description = "2023_06_23_raid_feature_updates",
+		.check = "SHOW COLUMNS FROM `raid_members` LIKE 'isassister'",
+		.condition = "empty",
+		.match = "",
+		.sql = R"(
+	ALTER TABLE `raid_members` 
+	ADD COLUMN `ismarker` TINYINT UNSIGNED DEFAULT(0) NOT NULL AFTER `islooter`,
+	ADD COLUMN `isassister` TINYINT UNSIGNED DEFAULT(0) NOT NULL AFTER `ismarker`,
+	ADD COLUMN `note` VARCHAR(64) DEFAULT("") NOT NULL AFTER `isassister`;
+
+	ALTER TABLE `raid_details` 
+	ADD COLUMN `markedNPC1` SMALLINT UNSIGNED DEFAULT(0) NOT NULL AFTER `motd`,
```

Comment:
`markedNPC1` should be `snake_case` or `marked_npc_1` see https://docs.eqemu.io/developer/conventions-guidelines/#column-naming

Reply:
Updated